src/cargo/lib.rs

```diff
@@ -150,6 +150,7 @@ use anyhow::Error;
 use log::debug;
 
 pub use crate::util::errors::{AlreadyPrintedError, InternalError, VerboseError};
+pub use crate::util::network::http;
```

Comment:
Why is this being brought to the top-level of the API?

Reply:
https://github.com/rust-lang/cargo/pull/12290/commits/24278ded7eb3da376521400a5ff1bdca48d0e189#r1235495481

Reply:
Because of this <https://github.com/rust-lang/cargo/pull/12290#discussion_r1235495481> 😞.